### ISSUE TITLE
Delay bossbar creation

### ISSUE THREAD

**Issue by h14turkiye** [toxicity188/BetterHud#37]
The hud commands doesn't remove the bossbar. Because of that I can't solve the conflicts with NEZNAMY TAB plugin. Delay bossbar creation so tab can initialize first, perhaps a configurable option for how much ticks to delay. I can't use merge-bossbar either because I use different fonts from better hud.
![image](https://github.com/user-attachments/assets/c9244b7a-7259-40af-ba85-4722c8230e4e)

**Comment by PSYCHEER** [comment]
I need that in TAB plugin for bossbars as I need BHUD to be first

**Comment by h14turkiye** [comment]
> I need that in TAB plugin for bossbars as I need BHUD to be first

Well, you can just use the console, /tab bossbar <player> actually removes the bossbar etc.

**Comment by PSYCHEER** [comment]
Yes I have some things done trough bossbars which are always on, on player join I need delay for them but yes, I will disable them and then enable them like a dumb... real solution
